host/3.0/buster/amd64/node/node10/node10-composite.template

```diff
@@ -0,0 +1,42 @@
+## Removed find /workers/node/grpc/src/node/extension_binary/ -mindepth 1 -type d ! -regex ".*linux-x64.*" -prune -exec rm -rf '{}' \;
+# This line removes grpc stuff built for unsupported x64 builds. Can probably delete from main host dockerfile.  need to inventory 
+
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
+ARG HOST_VERSION
+
+RUN apt-get update && \
+    apt-get install -y curl gnupg && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+    apt-get update && \
+    apt-get install -y nodejs
+
+# Chrome headless dependencies
+# https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix
+RUN apt-get install -y xvfb gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
+    libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 \
+    libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \
+    libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 \
+    libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
```

Comment:
except installing specific version of node, rest of the commands are common to all node images. This would be true for python as well -- all the common dependencies in python images except for the python version 

Does it make sense to have template per language as well?

Reply:
makes sense. Do track this separately - open issue / WI